Cargo.toml

```
@@ -43,7 +43,7 @@ arrayvec = "0.4.10"
 signal-hook = { version = "0.1.9", optional = true }
 
 [build-dependencies]
-nasm-rs = { git = "https://github.com/tdaede/nasm-rs", branch="msvc-OUT", optional = true }
+nasm-rs = { path = "nasm_rs/", optional = true }
```

```suggestion
nasm-rs = { version = "0.1", path = "nasm_rs/", optional = true }
```

This makes cargo publish happy.